konduit-serving-test/src/test/java/ai/konduit/serving/verticles/python/custom/PythonIntegerInputTest.java

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
Also, what's the difference between this class and `PythonIntegerInputOperationTest.java`?

Reply:
There is no much difference other than the python script doing some value update inside the python script to make sure that there is input and output values are working fine, 
as we did both scripts we kept both files .
please let us know if not needed we remove the normal files and keep only the operation based files.